homeassistant/helpers/selector.py

```diff
@@ -120,4 +120,4 @@ class TargetSelector(Selector):
     Value should follow cv.ENTITY_SERVICE_FIELDS format.
     """
 
-    CONFIG_SCHEMA = vol.Schema({"entity": {"domain": str}})
+    CONFIG_SCHEMA = vol.Schema({"entity": {"domain": str, "device_class": str}})
```

Comment:
Let's add these to the area selector too, as this can also picks an area

Reply:
Shouldn't that be implied ? I thought this config meant that we care about areas/devices that match this type of entity

Reply:
Yes, but it is weird we don't have it on the area selector and we do have it on the target selector while they can both select areas.

Reply:
I am sorry. I misunderstood. I thought you suggested we added an area config inside `target`. Will update.